### ISSUE TITLE
Sync namespaces for .NET Framework

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#61421]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Sync-namespaces-for-NET-Framework/10022724)._

---
Sync namespaces is only available for .NET Core projects.  Please add this functionality to .NET Framework projects, too.

---
#### Feedback Bot on 4/26/2022, 00:08 AM: 

(private comment, text removed)

**Comment by sharwell** [comment]
💡 This feature does not disable itself based on target framework, but it's only implemented for CPS projects. Projects that still use the legacy project system don't implement the capability support we rely on here:

https://github.com/dotnet/roslyn/blob/2fd58951039e953cbd5a3735c8eb8426d35a4cfe/src/VisualStudio/Core/Def/SyncNamespaces/SyncNamespacesCommandHandler.cs#L74-L75

**Comment by CyrusNajmabadi** [comment]
@sharwell should we move this then to the legacy project system owners so they can track these requests?

**Comment by sharwell** [comment]
@CyrusNajmabadi I don't believe there's any expectation of the legacy project system supporting these features. There are other ways to detect a legacy .NET/C# project, and we can rely on those at the above Roslyn code location.

**Comment by janseris** [comment]
.NET Framework projects really need this feature. And they need it more than .NET Core project just for the fact that they are legacy. This means that there is monolithic architecture etc. If someone is to fix an old project, this is one of the essential functionalities among with rich code navigation (number of references etc.).

**Comment by sharwell** [comment]
> .NET Framework projects really need this feature.

.NET Framework projects already have this feature, as long as they are using the new SDK-style projects. The feature is only missing from projects that use the old project system.

With that said, the issue is now marked as **help wanted**. We would welcome a pull request that updates the line I mentioned to account for the possibility that a project may be using the old project system, provided the feature still works in that case.

**Comment by Seabizkit** [comment]
please add
